nativesoundsystem/src/main/jni/aaudio/AAudioManager.cpp

```diff
@@ -97,29 +119,31 @@ bool AAudioManager::createEngine(SoundSystem *soundSystem) {
 #endif
     engine.bitsPerSample_ = SampleFormatToBpp(engine.sampleFormat_);
 
-    StreamBuilder builder(engine.sampleChannels_,
-                          engine.sampleFormat_,
-                          AAUDIO_SHARING_MODE_SHARED,
-                          AAUDIO_DIRECTION_OUTPUT);
+    StreamBuilder builder;
+    engine.playStream_ = builder.CreateStream(
+            engine.sampleFormat_,
+            engine.sampleChannels_,
+            AAUDIO_SHARING_MODE_SHARED,
+            AAUDIO_DIRECTION_OUTPUT,
+            INVALID_AUDIO_PARAM,
+            dataCallback,
+            &engine);
 
-    engine.playStream_ = builder.Stream();
     assert(engine.playStream_);
 
     PrintAudioStreamInfo(engine.playStream_);
-
     engine.sampleRate_ = AAudioStream_getSampleRate(engine.playStream_);
+    engine.framesPerBurst_ = AAudioStream_getFramesPerBurst(engine.playStream_);
+    engine.defaultBufSizeInFrames_ = AAudioStream_getBufferSizeInFrames(engine.playStream_);
+    AAudioStream_setBufferSizeInFrames(engine.playStream_, engine.framesPerBurst_);
+    engine.bufSizeInFrames_ = engine.framesPerBurst_;
+
     aaudio_result_t result = AAudioStream_requestStart(engine.playStream_);
```

Comment:
When I add those lines, `result != AAUDIO_OK` and I've still logs `[AAudioManager] currentState=3 AAUDIO_STREAM_STATE_STARTING` in the audio loop...

So for now, I do not push this `wait`...